fixtures/run_as_non_root_psc_false_allowed_v1.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  name: run_as_non_root_false_allowed
+  labels:
+    apps: fakeSecurityContext
+    audit.kubernetes.io/allow-run-as-root: "Superuser privileges needed"    
+  namespace: fakeDeploymentRANR  
+spec:
+  securityContext:
+    runAsNonRoot: false
+  containers:
+  - name: fakeContainerRANR
+    resources: {}
+    securityContext: {}
+status: {}
```

Comment:
Please set your editor to include newline. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
Same for all other files that don't have it